### ISSUE TITLE
False positive on `extra` kwargsstory

### ISSUE THREAD

**Issue by kornicameister** [kornicameister/loguru-mypy#93]
Consider following piece of code:

```python
        logger.info(
            'Booting up worker',
            extra={'fdas_queue': queue_name},
        )
```

It is obviously flagged as:

```sh
[mypy] Not all arguments converted during string formatting  [str-format] [E] 
```

**Comment by kornicameister** [comment]
@Delgan  @ThibaultLemaire  I'd like to gather your opinion on that please ;-)
Would it be consider a good call or a false positive ?

**Comment by ThibaultLemaire** [comment]
It doesn't do what it seems, so I'd say it's good that it raises an error. The `extra` dict is intended to be passed by **kwargs, so this won't work with a sink configured to use `extra[queue]`. (Although the error doesn't exactly tell you all that)

The real problem though is that it will tell you the same for
```python
logger.info(
    'Booting up worker',
    queue=queue_name,
)
```

But I think I've already touched on that somewhere that I can't find back at this time.

**Comment by Delgan** [comment]
Technically it's correct code. Basically, I would never consider this a error within Loguru.
However, it may be interesting to warn Mypy users about possible misuse. Most of the time, it's indeed a mistake. Especially for people migrating from standard `logging`. As long as it's possible to disable the warning for the theoretically valid use case, it seems to be a good catch in general.
